lib/twilio/device.ts

```diff
@@ -1213,7 +1204,7 @@ class Device extends EventEmitter {
       this._publishNetworkChange();
     });
 
-    const play = (this._enabledSounds.incoming && !wasBusy)
+    const play = (this.audio?.incoming() && !wasBusy)
       ? () => this._soundcache.get(Device.SoundName.Incoming).play()
```

Comment:
It seems like this kind of logic could be placed in the `AudioHelper` class itself. Probably out of scope for this ticket though.